mappings/community/DX12340A-requirements-1309234876.json

```diff
@@ -10,6 +10,37 @@
     },
     "status": 200,
     "jsonBody": {
+      "licenceConditions": [
```

Comment:
You appear to have two "licenseConditions" in your JSON

Reply:
weird, my bad!